Podcast/PlayerControlsView.swift

```diff
@@ -76,54 +92,90 @@ class PlayerControlsView: UIView {
         addSubview(rightTimeLabel)
         
         playPauseButton = UIButton(frame: .zero)
-        playPauseButton.frame.size = playPauseButtonSize
-        playPauseButton.frame.origin = CGPoint(x: self.frame.width/2 - playPauseButtonSize.width/2, y: self.frame.maxY - 102)
         playPauseButton.adjustsImageWhenHighlighted = false
         playPauseButton.addTarget(self, action: #selector(playPauseButtonPress), for: .touchUpInside)
         addSubview(playPauseButton)
         
+        playPauseButton.snp.makeConstraints { make in
+            make.size.equalTo(playPauseButtonSize)
+            make.centerX.equalToSuperview()
+            make.top.equalTo(slider.snp.bottom).offset(playPauseButtonTopOffset)
+        }
+        
         forwardsButton = UIButton(frame: .zero)
-        forwardsButton.frame.size = skipButtonSize
-        forwardsButton.frame.origin = CGPoint(x: playPauseButton.frame.maxX + skipButtonSpacing, y: self.frame.maxY - skipButtonYInset)
-        forwardsButton.setBackgroundImage(#imageLiteral(resourceName: "player_skip_forward_icon"), for: .normal)
+        forwardsButton.setBackgroundImage(#imageLiteral(resourceName: "forward30"), for: .normal)
         forwardsButton.adjustsImageWhenHighlighted = false
         forwardsButton.addTarget(self, action: #selector(forwardButtonPress), for: .touchUpInside)
         addSubview(forwardsButton)
+        forwardsButton.snp.makeConstraints { make in
+            make.size.equalTo(skipButtonSize)
+            make.top.equalTo(slider.snp.bottom).offset(skipButtonTopOffset)
+            make.leading.equalTo(playPauseButton.snp.trailing).offset(skipForwardSpacing)
+        }
         
         backwardsButton = UIButton(frame: .zero)
-        backwardsButton.frame.size = skipButtonSize
-        backwardsButton.frame.origin = CGPoint(x: playPauseButton.frame.minX - skipButtonSpacing - skipButtonSize.width, y: self.frame.maxY - skipButtonYInset)
-        backwardsButton.setBackgroundImage(#imageLiteral(resourceName: "player_skip_backward_icon"), for: .normal)
+        backwardsButton.setBackgroundImage(#imageLiteral(resourceName: "back30"), for: .normal)
         backwardsButton.adjustsImageWhenHighlighted = false
         backwardsButton.addTarget(self, action: #selector(backwardButtonPress), for: .touchUpInside)
         addSubview(backwardsButton)
+        backwardsButton.snp.makeConstraints { make in
+            make.size.equalTo(skipButtonSize)
+            make.top.equalTo(slider.snp.bottom).offset(skipButtonTopOffset)
+            make.trailing.equalTo(playPauseButton.snp.leading).offset(-1 * skipForwardSpacing)
```

Comment:
why can't we just do an inset here instead of -1 * skipForwardSpacing

Reply:
![image](https://user-images.githubusercontent.com/20246620/31853923-3caa761c-b65f-11e7-90f0-dc927052591b.png)
this happens, not really sure why but the offset works better

Reply:
Do 0-skipForwardSpacing. Compiler probably optimizes this hopefully but better then multiply